src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -98,6 +100,7 @@ public static void WarmUpCacheInCurrentProcess(Project project)
 
         public static async Task<SerializableUnimportedExtensionMethods> GetUnimportedExtensionMethodsInCurrentProcessAsync(
             Document document,
+            SemanticModel? semanticModel,
```

Comment:
when is this null?

Reply:
➡️ See https://github.com/dotnet/roslyn/pull/71872#discussion_r1475251037